core/templates/core/404.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'core/base.html' %}
+
+{% block content %}
+<div class="h-80vh w-screen my-28 flex items-center">
+    <div class="container flex md:flex-row items-center justify-between px-20 text-gray-700">
```

Comment:
Add color to each element, because now this bottom text is to dark 
![image](https://github.com/Gwardia-Czapli/GwardiaHub/assets/88268063/f9791670-6a63-46b9-a16d-b8ff96cb28de)

```suggestion
    <div class="container flex md:flex-row items-center justify-between px-20">
```